docs/dev_building_from_source.rst

```diff
@@ -0,0 +1,47 @@
+
+==============================
+Building libCellML from Source
+==============================
+
+This document covers building libCellML from source.  It is assumed that you already have the codebase downloaded and ready for building.  The variable LIBCELLML_SRC shall be used to refer to the directory containing the LICENSE file for libCellML.
```

Comment:
Agreed with @nickerso. If at all possible, I would keep libCellML for the project name, but LibCellml in the code. If not, then I would also change my vote.